src/uproot/_dask.py

```diff
@@ -855,8 +884,9 @@ def _get_meta_array(
     if form_mapping is not None:
         form = form_mapping(form)
 
-    empty_arr = form.length_zero_array(
-        behavior=None if form_mapping is None else form_mapping.behavior
+    empty_arr = awkward.Array(
+        form.length_zero_array(highlevel=False),
+        behavior=None if form_mapping is None else form_mapping.behavior,
```

Comment:
Is this a bug-fix, independent of the new feature intended from this PR?

If so, I'll implement it in a separate PR.

Reply:
correct it is a bug fix

Reply:
Yes, it is. I'll make that PR.

Reply:
https://github.com/scikit-hep/uproot5/pull/892